<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameJoinDialog</class>
 <widget class="QDialog" name="GameJoinDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Join game</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="radioButtonAddress">
       <property name="text">
        <string>Enter Address:</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditAddress"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="radioButtonLan">
       <property name="text">
        <string>Search on local networ&amp;k:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="listViewLan">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBarLan">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>0</number>
       </property>
       <property name="value">
        <number>-1</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
